src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -170,15 +221,13 @@ const ChartsHolder = ({ projections, region, chartId }: ChartsHolderProps) => {
     scrollTo(metricRefs[metric].current);
   };
 
-  const ccviScores = useCcviForFips(region.fipsCode);
-
   // TODO(pablo): Create separate refs for signup and share
   return (
     <>
       <ChartContentWrapper>
         <LocationPageHeader
-          alarmLevel={projections.getAlarmLevel()}
-          stats={projections.getMetricValues()}
+          alarmLevel={alarmLevel}
```

Comment:
now LocationPageHeader has no dependencies on Projections \o/ and can get everything from the locationSummary.